app/javascript/components/enrollment/steps/ExposureInformation.js

```diff
@@ -428,8 +428,9 @@ class ExposureInformation extends React.Component {
               className="ml-1 d-inline"
               checked={!!this.state.current.patient.continuous_exposure}
               onChange={this.handleChange}
+              disabled={this.state.current.patient.isolation}
             />
-            <InfoTooltip tooltipTextKey="continuousExposure" location="right"></InfoTooltip>
+            <InfoTooltip tooltipTextKey={this.state.current.patient.isolation ? 'continuousExposureDisabled' : 'continuousExposure'} location="right" />
```

Comment:
Since the disable is on the input, I think it would be better to have the tooltip be attached to the disabled input.  I know that makes the implementation a bit more complicated but that's what we do with disabled inputs across the app

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1241/commits/de6e8f7a42d55d2a02c10112d137563d716be34b